### PR TITLE
chore: add example cluster definition for Azure Stack

### PR DESCRIPTION
**Pull request by honcao**
**Reason for Change**:

add example cluster definition for Azure Stack

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by CecileRobertMichon** [comment]
/approve

@jadarsie for review

File: examples/azurestack/kubernetes-azurestack-azure-ad.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
```
Comment (jadarsie) [inline examples/azurestack/kubernetes-azurestack-azure-ad.json]: Could you please run "format document" on these two?

Comment (honcao) [reply examples/azurestack/kubernetes-azurestack-azure-ad.json]: fixed

**Pull request merged**